EIPS/draft-eip-multi-algorithm-pki.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Support for transactions signed with non-secp256k1 algorithms
+description: This EIP adds a standardized way to support additional algorithms to signed transactions
```

Comment:
```suggestion
description: Introduces a new transaction type allowing alternative transaction signature algorithms
```

This still isn't perfect, but it's going in the right direction. Feel free to play with this and not take this suggestion literally. The main concern is that we want the description and title to be as information-dense as possible.